src/ModalComponent.js

```diff
@@ -29,8 +41,48 @@ class ModalComponent extends Component {
         <div className="modal-footer">
           <div className="footer-cage">
             <div className="footer-cage-text">
-              {this.props.data.fullName}{' '}
-              {this.props.data.position && this.props.data.position.label}...
+              <p style={signatureBodystyles}>
+                {fullName}
+                <br />
+                {position && position.label}
+                <br />
+                <b>{title}</b>
+                <br />
+                <a href="">{email} </a>
+                <br />
+                {Parser(presentNumbers(phoneNumbers))}
+                Skype ID: {skypeId} <br />
+                <a href="http://www.belatrixsf.com">
+                  http://www.belatrixsf.com
+                </a>
+                <br />
+                <br />
+              </p>
+
+              <p style={signatureDisclaimerStyles}>
+                <u>WARNING OF CONFIDENTIALITY</u>: The information contained and
```

Comment:
Please reuse the existing function for these content.

Reply:
Puedo traer el contenido del disclaimer de otro lado es decir como encapsular todo ese texto pero aun no el getConfidentialityWarning() ya que la firma del preview y la firma final tienen segun Fiorella diferentes tamaños, altura de linea etc. Voy a consultarle de nuevo en caso me digan q pueden tener ambos el mismo estilo ahi si ya podria usar la misma funcion para todos los componentes.